docs/search-configuration/custom-dictionaries.md

```diff
@@ -0,0 +1,74 @@
+# Custom User Dictionaries
+
+The search module supports 3 types of custom dictionaries for helping to tune search results to your content. These are synonyms, stop words and custom text analysis for Japanese.
+
+In the website admin you can upload text files (`.txt` only) or in the case of stop words and synonyms manually enter them on the [Search Configuration](admin://admin.php?page=search-config) screen.
+
+Note that only a file upload is supported for Japanese user dictionaries.
```

Comment:
Interesting; why?

Reply:
The file upload and the manual entry count as separate files, this is easy to support for stopwords and synonyms but for the user dictionary there can only be a single tokeniser and a single user dictionary file in the settings for it. I can add that information to the docs here.